app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -24,6 +24,7 @@ def create
         form: @form,
         save_service: SaveProviderUser.new(provider_user: provider_user),
         invite_service: InviteProviderUser.new(provider_user: provider_user),
+        new_user: !@form.persisted?,
```

Comment:
Would `form.existing_provider_user.blank?`(or words to that effect) work?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1889/commits/f19c8a5022b2418410a5996a50b257ce80f2ddf5